src/main/java/stringCalculator/domain/ArithmeticOperator.java

```diff
@@ -0,0 +1,33 @@
+package stringCalculator.domain;
+
+import java.util.Arrays;
+import java.util.function.IntBinaryOperator;
+
+
+public enum ArithmeticOperator {
+
+    PLUS("+", (firstNumber, secondNumber) -> firstNumber + secondNumber),
+    MINUS("-", (firstNumber, secondNumber) -> firstNumber - secondNumber),
+    MULTIPLY("*", (firstNumber, secondNumber) -> firstNumber * secondNumber),
+    DIVIDE("/", (firstNumber, secondNumber) -> firstNumber / secondNumber);
+
+    private final String operator;
+    private final IntBinaryOperator experssion;
```

Comment:
- `IntBinaryOperator` 활용 👍
- 변수 이름에 오타가 있는 것 같습니다.
```suggestion
    private final IntBinaryOperator expression;
```